### PR TITLE
Make October 2024 news public

### PR DESCRIPTION
**Pull request by cticenhour**
Adds November 2024 template and also makes a few small adjustments to grammar, whitespaces, etc. 

@idaholab/moose-ccb Please add your contributions, as well as those that you have reviewed or been affiliated with, if those are not in the news already. Thanks!

Refs #11447

### CODE REVIEW

File: modules/doc/content/newsletter/2024/2024_10.md [hunk 1]
```diff
@@ -1,12 +1,64 @@
...
+to one or the other depending on the objects at hand. For example, a Dirichlet boundary condition for a Lagrange variable
+will use a nodeset for its boundary parameter, while a postprocessor computing the diffusive flux with a monomial variable
+will use a sideset for its boundary parameter. As such, we focused our mesh generation effort to be able to easily create sidesets,
+and the nodesets can be easily generated from these sidesets.
+
+However, for the specific need of generating 1D nodesets within a 3D mesh, we needed to be able to generate nodesets directly.
+The [ParsedGenerateNodeset.md] joins the roster of mesh generators for that purpose. It offers much of the flexibility of
+the [ParsedGenerateSideset.md] and their design are very similar.
+
+### Mesh conversion to all simplices (triangles and tets)
```
Comment (GiudGiud) [inline modules/doc/content/newsletter/2024/2024_10.md:27]: @roystgnr good enough?

File: modules/doc/content/newsletter/2024/2024_10.md [hunk 2]
```diff
@@ -1,12 +1,64 @@
...
+
+The [ElementsToSimplicesConverter.md] was created as a mesh generator. An input mesh will be modified
+to replace each non-simplex mesh element with a set of simplices
+connecting the same nodes.  Each quad is split into 2 triangles, each
+pyramid into 2 tetrahedra, each prism into 3 tets, and each cube into
+6 tets.
+
+### Ability to set multiple time integrators
```
Comment (GiudGiud) [inline modules/doc/content/newsletter/2024/2024_10.md:35]: @lindsayad  good enough?

File: modules/doc/content/newsletter/2024/2024_10.md [hunk 3]
```diff
@@ -1,12 +1,64 @@
...
+
+For simulations using multiple equations, notably in the context of solving multiple nonlinear
+systems, there is now the option to use different time integration methods for each equation.
+
+### New tests scheduler for High Performance Computing
```
Comment (GiudGiud) [inline modules/doc/content/newsletter/2024/2024_10.md:40]: @loganharbour  good enough?

File: modules/doc/content/newsletter/2024/2024_10.md [hunk 4]
```diff
@@ -1,12 +1,64 @@
...
+
+### Navier Stokes: Skewness correction changes
+
+The gradient of the pressure, both corrected and uncorrected, that are used in Rhie-Chow interpolation are now both
+using skewness correction when skewness correction has been specified for the pressure variable.
+
+Skewness-corrected gradients are now computed using the Least-Squares method rather than Green Gauss.
+With these two modifications, oscillations on skewed meshes in simulations with volumetric body forces are no longer present.
+
+## Thermal hydraulics: use of field variables on node elements for Junctions
```
Comment (GiudGiud) [inline modules/doc/content/newsletter/2024/2024_10.md:55]: @joshuahansel  good enough?

File: modules/doc/content/newsletter/2024/2024_10.md (hunk 1 above)
Comment (cticenhour) [reply modules/doc/content/newsletter/2024/2024_10.md:27]: Ping @roystgnr

Comment (roystgnr) [reply modules/doc/content/newsletter/2024/2024_10.md:27]: Looks good to me; thanks very much for adding it.

File: modules/doc/content/newsletter/2024/2024_10.md [hunk 5]
```diff
@@ -1,12 +1,64 @@
...
+The [ElementsToSimplicesConverter.md] was created as a mesh generator. An input mesh will be modified
+to replace each non-simplex mesh element with a set of simplices
+connecting the same nodes.  Each quad is split into 2 triangles, each
+pyramid into 2 tetrahedra, each prism into 3 tets, and each cube into
+6 tets.
+
+### Ability to set multiple time integrators
+
+For simulations using multiple equations, notably in the context of solving multiple nonlinear
+systems, there is now the option to use different time integration methods for each equation.
```
Comment (lindsayad) [inline modules/doc/content/newsletter/2024/2024_10.md:38]:
```suggestion
For simulations using multiple variables/equations there is now the option to use different time integration methods for each variable/equation.
```

File: modules/doc/content/newsletter/2024/2024_10.md (hunk 2 above)
Comment (lindsayad) [reply modules/doc/content/newsletter/2024/2024_10.md:35]: good now

**Review by milljm — APPROVED** [review]
💯

**Pull request merged**